lms/static/sass/elements/_program-card.scss

```diff
@@ -89,8 +97,30 @@
   }
 
   .certificate-status {
-    font-size: em(12);
-    color: $gray;
+    .link {
+      font-size: em(12);
+      color: $gray;
+    }
+ 
+    .secondary {
+      @extend %hide-until-focus;
+    }
+
+    .link {
```

Comment:
@marcotuts I am styling these to remain the same as the original design for progress text. They are links for tab focus, but users will not be able to hover the cursor over them.